recipes/secimtools/meta.yaml

```diff
@@ -13,46 +13,42 @@ source:
   '{{ hash_type }}': '{{ hash_value }}'
 
 build:
-  number: 0
+  number: 1
```

Comment:
@moskalenko I would rather but an rc version string into the real package version. The way you are doing it here will give you a new version even if you specify 1.0.0.

Reply:
Can you clarify the above statement? Thank you.

Reply:
If you type in `conda install secimtools=1.0.0` you are expecting one single version with a specific set of features and bugs. You are keeping this version number but increasing the build number and updating the source. This means the version 1.0.0 has now potential new bugs and features. So it is not reproducible for people that want to install the old version 1.0.0.

Reply:
I really don't want to settle on a 1.0.0 or set any expectations of reproducibility until the paper is accepted at which time I will release 1.0.0 packages and use semantic versioning from there on. This is pre1.0.0 with minor fixes going in to get the galaxy wrappers done, which is the main blocker at this time. Wrappers need this update.

Reply:
In this case should I remove the older version?

Reply:
Sure. Danke.